apps/worker/src/app/shared/shared.module.ts

```diff
@@ -127,15 +149,18 @@ const PROVIDERS = [
   },
   InvalidateCacheService,
   CreateLog,
-  {
-    provide: WsQueueService,
-    useClass: WsQueueService,
-  },
   {
     provide: StorageService,
     useClass: getStorageServiceClass(),
   },
+  QueueServiceHealthIndicator,
+  TriggerQueueServiceHealthIndicator,
+  WsQueueServiceHealthIndicator,
+  QueueService,
+  TriggerQueueService,
+  WsQueueService,
   StorageHelperService,
+  readinessService,
```

Comment:
Order matters. If a provider is dependant of other providers, needs to be added later in the array. Learnt it the hard way. 🙁